hack/build_etcd.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+if [  $# -gt 0 ]
+    then
+    ETCD_VERSION="$1";
+    else
+    ETCD_VERSION="master";
+fi
+
+echo "Using ETCD version $ETCD_VERSION"
+
+mkdir /hack
```

Comment:
for me this fails. Shouldn't the `hack` directory be created inside the workdir instead of the root?